spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -119,8 +120,15 @@ public InternalRow[] call(InternalRow args) {
       checkDuplicateFiles = args.getBoolean(3);
     }
 
+    int parallelism;
```

Comment:
I'm not sure how we ended up with this pattern here, in the other procedures we do
```java
var = (arg.nullat(?)) default : arg.get(?);
```
Although There is probably no reason to change this now. I do think a default of 1 is still probably too low. I feel like this is a `gotcha` default and is basically never correct.

Reply:
So we set default here to worker pool default thread count too?

Reply:
@RussellSpitzer , Thanks for your review, I have fixed it to `ThreadPools.WORKER_THREAD_POOL_SIZE`